conf/machine/nitrogen8m.conf

```diff
@@ -11,10 +11,15 @@ require conf/machine/include/arm/arch-arm64.inc
 
 # Kernel configuration
 PREFERRED_PROVIDER_virtual/kernel ??= "linux-boundary"
+PREFERRED_VERSION_linux-boundary = "4.14%"
```

Comment:
Why?

Reply:
We have support for our 4.14 kernel in meta-boundary and we wish to AUTOREV and use our own boundary kernel defconfig.